HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -129,11 +129,26 @@ def customiseFor2018Input(process):
 
     return process
 
+def customiseForPixelCPE(process):
```

Comment:
@OzAmram I think it's customary to call this customizations with the PR number,  e.g. `customizeFor33495`.
@Martin-Grunewald can comment if this is acceptable.

Reply:
> @OzAmram I think it's customary to call these customisations with the PR number, e.g. `customiseFor33495`.
> @Martin-Grunewald can comment if this is acceptable.

@OzAmram please add the PR number as suggested by @mmusich : if you do so, once the PR will get merged, it will be easier to @Martin-Grunewald to identify the customization that can be removed

Reply:
Ok I just updated the name